packages/orchestrator/lib/clients/client.integration.test.ts

```diff
@@ -33,6 +34,7 @@ describe('OrchestratorClient', async () => {
 
     afterAll(async () => {
         scheduler.stop();
+        await setTimeout(1000); // waiting for the scheduler to stop
```

Comment:
I wish I didn't need this but the scheduler internals relies on messages passing to the worker_thread which is asynchronous